<launch>
  <arg name="enable_color" default="true"/>
  <arg name="enable_infra1" default="false"/>
  <arg name="enable_infra2" default="false"/>
  <arg name="enable_imu" default="false"/>
  <arg name="enable_fisheye" default="false"/>
  <arg name="enable_depth" default="false"/>
  <arg name="color_fps" default="60"/>
  <arg name="enable_pointcloud" default="false"/>

    <node name="aruco_detect" pkg="agv_detect" type="aruco_detect" output="screen">
        <param name="squareLength" type="double" value="0.062"/>
        <param name="markerLength" type="double" value="0.0496"/>
        <param name="showRejected" type="string" value=""/>
        <param name="estimatePose" type="string" value="/home/godssi/catkin_ws/src/agv_detect/include/calibration_custom.yml"/>
        <param name="autoScale" type="string" value=""/>
        <param name="markerDistance" type="double" value="1.8"/>
        <param name="multipleMarkers" type="bool" value="false"/>
        <param name="detectorParams" type="string" value=""/>
        <param name="refine" type="int" value="1"/>
        <param name="camId" type="int" value="4"/>
        <param name="video" type="string" value=""/>
        <param name="dictionary" type="int" value="3"/>
        <param name="custom_dictionary" type="string" value=""/>
    </node>
    
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
   	  <param name="video_device" value="/dev/video4" />
      <param name="image_width" value="960" />
      <param name="image_height" value="540" />
      <param name="framerate" value="60" />
      <param name="pixel_format" value="yuyv" />
      <param name="color_format" value="yuv422p" />
      <param name="camera_frame_id" value="usb_cam" />
    </node>
</launch>
